.idea/fun-react-lib.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this an IDE workspace file?

Reply:
Good catch, this should not be in there. Can you change this?

Reply:
@Nicxzmiller, you're currently working on removing this?